public/components/layer_config/documents_config/document_layer_source.tsx

```diff
@@ -394,10 +402,12 @@ export const DocumentLayerSource = ({
           <EuiFlexGrid columns={1}>
             <EuiFlexItem>
               <EuiCheckbox
-                id="toggle-tooltip"
-                label="Show tooltips"
-                checked={showTooltips}
-                onChange={onShowTooltipsChange}
+                id="enable-tooltip"
+                label={i18n.translate('documentLayer.enableTooltips', {
+                  defaultMessage: 'Enable tooltips',
+                })}
+                checked={enableTooltips}
```

Comment:
Can we replace here with `selectedLayerConfig.source.showTooltips` directly? So that we don't need the enableTooltips useState hook.

Reply:
Yeah. i noticed that too. Will do this in next PR as refactoring.